src/components/WebexSettings/WebexSettings.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {
```

Comment:
is it possible to make this a single line?

Reply:
Linter would not allow it